business-registry-dashboard/app/components/Table/AffiliatedEntity/index.vue

```diff
@@ -42,22 +42,38 @@ watch(
     <!-- columns to show dropdown -->
     <template #header-right>
       <USelectMenu
-        v-slot="{ open }"
         v-model="affStore.selectedColumns"
         :options="affStore.optionalColumns"
+        option-attribute="label"
         multiple
-        :ui="{ trigger: 'flex items-center w-full' }"
+        selected-icon="hidden"
         @change="affStore.setColumns"
       >
-        <UButton
-          color="white"
-          class="h-[42px] flex-1 justify-between text-gray-700"
-          :aria-label="$t('btn.colsToShow.aria', { count: affStore.selectedColumns.length })"
-        >
-          <span>{{ $t('btn.colsToShow.label') }}</span>
+        <template #default="{ open }">
```

Comment:
The select menu was in a very bad state. Now it looks a loot nicer:
Before:
![image](https://github.com/user-attachments/assets/51391aae-97b5-4892-8047-a9679fd3df34)
Now:
![image](https://github.com/user-attachments/assets/4ed5e3e1-7153-42e2-a3cd-dc9f8bc97c5f)

We're following now BTR design: https://dev.directorsearch.bcregistry.gov.bc.ca/

Reply:
It still looks like Eve from WALL-E (head detached from the body) 😆

Reply:
Hahahahaha, I can totally see it 😆 
![c6b91d47538de8534d5302bdb6135eb0](https://github.com/user-attachments/assets/9e571d61-d63e-4ef7-83e4-6220123984fd)